apps/condo/domains/miniapp/components/AppIndex/IndexBillingAppPage.tsx

```diff
@@ -57,23 +59,39 @@ export const IndexBillingAppPage: React.FC<IndexBillingAppPageProps> = ({ id })
     }
 
     const PageTitle = get(integration, 'name', BillingMessage)
+
+    if (PageTitle && title === BillingMessage) {
+        setTitle(PageTitle)
```

Comment:
Change state inside render block of component without any hooks looks like potential unnecessary render

Reply:
Refactored 😄 👍